static.go

```diff
@@ -237,6 +240,37 @@ const (
 				</form>
 			</div>
 		</section>
+		<section class="row">
+			<div id="restore" class="col-md-6">
+				<h3>Restore</h3>
```

Comment:
I suppose if it becomes an issue, we can always fix the UI.

The RO UI requires knowing a lot about how RO works. We should not continue to develop it in this direction, but rather should make new features (and old features!) better at being self-documenting.